### PR TITLE
Update to nx 18.0.4 and Angular 17.2.1

### PR DESCRIPTION
**Pull request by matheo**
FYI @DominikPieper

### CODE REVIEW

**Review by eduardoRoth — APPROVED** [review]
Great work!

File: packages/ionic-react/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nxext/ionic-react",
-  "version": "17.0.0",
+  "version": "18.0.4",
```
Comment (matheo) [inline packages/ionic-react/package.json:3]: @eduardoRoth @DominikPieper I bumped the version manually as I didn't get how the release works :wink:

Comment (DominikPieper) [reply packages/ionic-react/package.json:3]: Yes that’s fine

Comment (matheo) [reply packages/ionic-react/package.json:3]: @DominikPieper feel free to merge it :)

**Pull request merged**